### ISSUE TITLE
Removing StackableItem and making the feature of stack-ability existed in Item by default

### ISSUE THREAD

**Issue by BenCheung0422** [MinicraftPlus/minicraft-plus-revived#432]
**Is your feature request related to a problem? Please describe.**
As the stack-ability has made many exceptions on stacking, adding, removing, etc. Combining the feature altogether would make the procedure simpler.

**Describe the solution you'd like**
Removing the whole `StackableItem` class and put all the corresponding members into `Item` originally. Simplifying all the procedures relying on the stack-ability (checking if the item is instance of `StackableItem`).

**Comment by KalmeMarq** [comment]
Yes, but use an ItemStack. An Item is a just definitions and stuff. What you want is a Stack of a certain Item.

**Comment by BenCheung0422** [comment]
Right, the definitions of objects and the instances of objects should be separated into different class/handlers.
